pangea-sdk/service/url_intel/service.go

```diff
@@ -0,0 +1,22 @@
+package url_intel
+
+import (
+	"context"
+
+	"github.com/pangeacyber/pangea-go/pangea-sdk/pangea"
+)
+
+type Client interface {
+	Lookup(ctx context.Context, input *UrlLookupRequest) (*pangea.PangeaResponse[UrlLookupResult], error)
+}
+
+type UrlIntel struct {
+	*pangea.Client
+}
+
+func New(cfg *pangea.Config) *UrlIntel {
```

Comment:
In general, the UrlIntel struct should be private and New should return a Client...